docs/src/pages/lab/breadcrumbs/SimpleBreadcrumbs.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Breadcrumbs from '@material-ui/lab/Breadcrumbs';
+import Breadcrumb from '@material-ui/lab/Breadcrumb';
+import Link from '@material-ui/core/Link';
+
+const styles = theme => ({
+  root: {
+    padding: `${theme.spacing.unit}px ${theme.spacing.unit * 2}px`,
+  },
+});
+
+function handleClick(event) {
+  event.preventDefault();
+  alert('You clicked a breadcrumb.'); // eslint-disable-line no-alert
+}
+
+function SimpleBreadcrumbs(props) {
+  const { classes } = props;
+  return (
+    <Paper className={classes.root}>
+      <Breadcrumbs arial-label="Breadcrumb navigation">
+        <Breadcrumb>
+          <Link color="inherit" href="/" onClick={handleClick}>
```

Comment:
I think @eps1lon’s suggestion was that these should link to the demo title, rather than navigate away.

Reply:
We prevent the navigation.